### DISCUSSION TITLE
想问一下，Release Notes 是如何生成的呢

### DISCUSSION THREAD

**Discussion by EkkoG** [shadowsocks/shadowsocks-rust#1550] [General]
https://github.com/shadowsocks/shadowsocks-rust/releases

日志看起来很规整，像是自动化工具生成的，但是在项目中没有找到相关代码，好奇问一下

**Comment by zonyitoo** [comment]
Mostly written by hand. Some of them are generated by [Refined Github](https://github.com/refined-github/refined-github).
